src/controllers/admin.controller.js

```diff
@@ -70,18 +61,14 @@ adminCtrl.renderEditCourseForm = async (req, res) => {
         let responseCourses = await universityServiceAPI.getAllCourses();
         courseDetails = responseCourses.data.filter(function (c) { return c._id == courseId; });
 
-        if (courseDetails.length == 0){
```

Comment:
Why did you remove this validation?

Reply:
I updated again, I was trying something with it but forgot to put it again